### PR TITLE
Add automatic deployment using GH actions

### PR DESCRIPTION
**Pull request by dingo-d**
Added GH Actions deployment workflow

### CODE REVIEW

**Review by iruzevic — APPROVED** [review]

**Comment by dingo-d** [comment]
Not sure if these components should be removed or not, I've branched out of the main branch 🤷🏼‍♂️

**Review by IvanGrginovInf — APPROVED** [review]

**Comment by gabriel-glo** [comment]
@dingo-d I don't think they should be removed. Is the site working and looking fine when you start the build process?

**Comment by dingo-d** [comment]
yeah, the build fails to compile. Not sure why, I've tried rebasing the main branch 🤷🏼‍♂️

File: .github/workflows/documentation.yml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+          ssh-private-key: ${{ secrets.GH_PAGES_DEPLOY }}
+      - name: Release to GitHub Pages
+        env:
+          USE_SSH: true
+          GIT_USER: git
+        run: |
+          git config --global user.email "actions@github.com"
+          git config --global user.name "gh-actions"
+          if [ -e yarn.lock ]; then
+            yarn install --frozen-lockfile
```
Comment (IvanGrginovInf) [inline .github/workflows/documentation.yml:51]: why yarn?

Comment (dingo-d) [reply .github/workflows/documentation.yml:51]: Just default docusaurus build. If there is no `yarn.lock` it will default to `npm` 🙂

**Review by goranalkovic-infinum — APPROVED** [review]

**Review by gabriel-glo — APPROVED** [review]

**Pull request merged**